### PR TITLE
changes back readme

### PR DESCRIPTION
**Pull request by BushraAks**
I changed back the readme file to the original state

### CODE REVIEW

**Comment by mutasim77** [comment]
readme is cute, isn't it? 😁

File: README.md [hunk 1]
```diff
@@ -1,3 +1 @@
 # Quotely 🖋
-
-hello cute onee
```
Comment (BushraAks) [inline README.md:3]: u changed it back? 😂

Comment (BushraAks) [reply README.md:3]: waittt nevermind, i confusedd

**Comment by BushraAks** [comment]
> readme is cute, isn't it? 😁

so random this question, got me confused 😂, u didnt make any changes did u?

**Comment by BushraAks** [comment]
I cannot see the changes, only the ones i made

**Pull request merged**